shared/src/main/scala/drt/shared/redlist/LhrRedListDates.scala

```diff
@@ -0,0 +1,13 @@
+package drt.shared.redlist
+
+import drt.shared.CrunchApi.MillisSinceEpoch
+
+trait LhrRedListDates {
+  val t3RedListOpeningDate: MillisSinceEpoch
+  val t4RedListOpeningDate: MillisSinceEpoch
+}
+
+case object LhrRedListDatesImpl extends LhrRedListDates {
```

Comment:
This is an interesting one. After you questioned the use of 'Like' for interfaces/traits a while ago I tried to see what the generally accepted convention is for naming these things. One guy pointed out that your code should be written to interfaces, so to avoid scattering 'Like' everywhere for function arguments, you should instead name the production code's implementation with an '...Impl'